### ISSUE TITLE
NH-Haze dataset

### ISSUE THREAD

**Issue by chihlingchang** [c-yn/FocalNet#4]
Hi,
I’m currently trying to train the model on NH-Haze dataset for image dehazing with the given code, and I had followed the settings showed in the Supplementary Material including setting training epoch=5000, batch size=2, initial learning rate=2e-4, cropping the images into 800*1200, and splitting the dataset into 45 training data, 5 validation data and 5 testing data.

However, my testing result (PSNR=19.78) is not as good as the performance shows in the paper (PSNR=20.43).
I’d like to confirm whether there is any setting that I misunderstood, leading to the difference between our results?

Thank you so much!!

**Comment by c-yn** [comment]
Thanks for the question. We use 50 (45 training+5 validation) paired images for training following DeHamer.

**Comment by chihlingchang** [comment]
Thanks for your reply!

**Comment by chihlingchang** [comment]
Hi,
I had tried to add the 5 validation paired images into training dataset, however, my testing PSNR is 20.06. I’m wondering if there is any other skills that I had missed can help to enhance the performance?
Thank you!!
